scripts/configFromDB.py

```diff
@@ -155,7 +155,8 @@
     if len(missions) > 1:
         raise NotImplementedError("Can't yet handle multi mission db")
     mission = missions[0]
-    mission = dbu.getEntry('Mission', mission)
+    mission_id = dbu.getMissionID(mission)
+    mission = dbu.getEntry('Mission', mission_id)
```

Comment:
This is again an issue of getEntry needing to be fixed; this code shouldn't have to change.

Reply:
Agree.